content/topics/bridging-course-hard-skills/oop-concepts/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Object Oriented Programming concepts
+ready: True
+---
+
+### OOP concepts
```

Comment:
This heading is redundant redundant

Pleas look at the page when it gets rendered. If the text is repetitive it's not good